diff --git a/dist/chunk-WMQQYH7W.js b/dist/chunk-WMQQYH7W.js
index bbff431156e196eca46bd294a7cbe38a0eb874e9..4fc097050b785323e3276aaff30bbae4f8c17656 100644
--- a/dist/chunk-WMQQYH7W.js
+++ b/dist/chunk-WMQQYH7W.js
@@ -40,7 +40,8 @@ async function* parseSseStream(response) {
       } else if (line.startsWith("event:")) {
         eventType = line.substring("event:".length).trim();
       } else if (line.startsWith("data:")) {
-        eventData = line.substring("data:".length).trim();
+        const chunk = line.substring("data:".length).trim();
+        eventData = eventData ? eventData + "\n" + chunk : chunk;
       }
     }
   }
